ssas/service/api_common.go

```diff
@@ -0,0 +1,23 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/CMSgov/bcda-ssas-app/ssas"
+	"net/http"
+)
+
+func JsonError(w http.ResponseWriter, errorStatus int, errorText string, description string) {
+	fallbackMessage := fmt.Sprintf(`{"error": "%s", "error_description": "%s"}`, http.StatusText(http.StatusInternalServerError), http.StatusText(http.StatusInternalServerError))
+	e := ssas.ErrorResponse{Error: errorText, ErrorDescription: description}
+	body, err := json.Marshal(e)
+	if err != nil {
+		http.Error(w, fallbackMessage, http.StatusInternalServerError)
+	}
+	ssas.Logger.Printf("%s; %s", description, errorText) // TODO: log information about the request
```

Comment:
Should this `TODO` remain in the code for future work?  Or is it something that was missed as part of this ticket?

Reply:
@msnook Future work: I've created a [new ticket](https://jira.cms.gov/browse/BCDA-2784).  This PR's refactoring hasn't intentionally changed anything about the logs.

Reply:
Cool, thanks for making a ticket.